src/OpenTelemetry/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+## 1.3.2
+
+Released 2022-Dec-20
+
+* Fix Histogram synchronization issue: Use the same synchronization mechanism
+  for Histograms Update and Snapshot.
+  ([#4031](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4031))
```

Comment:
Maybe provide a bit more context on why we're patching this into an older release

```suggestion
* Support for metrics was originally released in version 1.3.0. A synchronization issue with regards to the histogram aggregation was discovered and fixed in [#3534](https://github.com/open-telemetry/opentelemetry-dotnet/pull/3534). This fix has been back-ported to OpenTelemetry .NET 1.3.2. If you are currently running OpenTelemetry .NET version 1.3.x, it is recommended that you upgrade to 1.3.2.
  ([#4031](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4031))
```